docs/contribute.rst

```diff
@@ -75,6 +75,9 @@ via::
 
     invoke tests
 
+(For more on invoke, see `invoke -l` to list tasks, or
+[pinvoke.org](pyinvoke.org) for documentation.)
```

Comment:
This is reST, not Markdown, so you'll need double-ticks. Also, I'd include the scheme. In short:
```
(For more on Invoke, see ``invoke -l`` to list tasks, or
[pyinvoke.org](https://pyinvoke.org) for documentation.)

Reply:
Oops. Because it's reST, you'll also need to change the link syntax.

Reply:
Ah thanks. Changed to accept your comment, and look okay in rST (the `[pyinvoke.org]` didn't work).